CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -238,6 +246,22 @@ protected function processThrows(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
      */
     protected function processParams(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $commentStart)
     {
+        if ($this->_platform === null) {
```

Comment:
All of this can be moved to sniff constructor to avoid checking each time if platform was set, because neither PHP version nor CLI values change during single run.